backend/src/main/kotlin/com/tngtech/apicenter/backend/config/SecurityConfiguration.kt

```diff
@@ -21,22 +21,27 @@ class SecurityConfiguration : WebSecurityConfigurerAdapter() {
 
     override fun configure(httpSecurity: HttpSecurity) {
         httpSecurity
-            .cors()
-            .and()
-            .csrf().disable()
-            .authorizeRequests()
-            .antMatchers("/sessions").permitAll()
-
-            .antMatchers("/").permitAll()
-            .antMatchers("/3rdpartylicenses.txt").permitAll()
-            .antMatchers("/favicon.ico").permitAll()
-            .antMatchers("/index.html").permitAll()
-            .antMatchers("/*.js*").permitAll()
-            .antMatchers("/open-iconic.*").permitAll()
-            .anyRequest().authenticated()
-            .and()
-            .addFilter(JwtAuthorizationFilter(authenticationManager(), jwtSecuritySecret))
-            .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+                .cors()
+                .and()
+                .csrf().disable()
+                .authorizeRequests()
+                .antMatchers("/api/1.0/sessions").permitAll()
```

Comment:
There's lots of different matchers. Do we really want the `Ant` ones, or is there something better?

Reply:
I was deliberately overcautious here, because (afaiu) the Ant matchers match resources from a number of different roots: `/resources`, `/resources/public`, and the HTTP routes. If you're more liberal with the permitted routes, you can access the properties files, for example.